.env.sample

```diff
@@ -5,3 +5,5 @@ DB_NAME= name of your database
 DB_HOST= name of your database
 DB_PORT= name of your database
 TESTDB_NAME= name of your test database
+APP_EMAIL=noreply@riders-ah-backend.com
+APP_LINK=htts://riders-ah-backend.herokuapp.com
```

Comment:
I suggest removing the actual values of a APP_EMAIL and APP_LINK from your .env.sample file and add just the use of each environment variable. and also add the SENDGRID_API_KEY env variable.